05-jupyter-ecosystem.md

```diff
@@ -141,8 +142,11 @@ single or double `%`, but some cannot.
      static images embedded in the notebook, and `%matplotlib
      notebook` produces interactive images (with zooming, panning, etc.).
      
-  *  The `%run` magic allows running external scripts, captures output and
-     displays it in the notebook, e.g., `%run my_script.py`.
+  * The `%run` magic allows running external scripts (and other
+     notebooks), captures output and displays it in the notebook,
+     e.g., `%run my_script.py`. The `%run` magic is one answer to "how
+     do I import one notebook into another?" (although there are some
+     differences between importing and running).
```

Comment:
Could you edit out the syntactic expletive "there are"?
Suggest: "(although importing and running are distinct actions)

Reply:
This comment is really about the differences between "the idea of importing" versus "the idea of running" because there is no way to import a notebook (as an action) . In other words, if you saved the notebook as a script, and imported that, there would be some minor differences compared to `%run notebook.ipynb`. But that is too complicated, and most people wouldn't appreciate that difference.

Reply:
Maybe you can explain that. 

I was just complaining about the grammatical clutter ("there are").

Reply:
Maybe we should just delete the parenthetical? Is it too complicated to explain the difference in this bullet point?